lib/fileutils.rb

```diff
@@ -1017,13 +1061,16 @@ def rm_r(list, force: nil, noop: nil, verbose: nil, secure: nil)
   end
   module_function :rm_r
 
+  # Equivalent to:
   #
-  # Equivalent to
+  #   FileUtils.rm_r(list, force: true, **kwargs)
   #
-  #   FileUtils.rm_r(list, force: true)
+  # May cause a local vulnerability if not called with keyword argument
+  # <tt>secure: true</tt>.
```

Comment:
Can you add a link to the place that explains this vulnerability?

Reply:
I'll be reworking all about secure and vulnerabilities, putting it into the doc for the module, and adding links as needed.